pages/apim/3.x/how-tos/how-to-enable-opentracing-with-jaeger.adoc

```diff
@@ -7,17 +7,96 @@
 :page-layout: apim3x
 :page-toc: false 
 
-== Why Enable OpenTracing? 
+== Why Should I Enable OpenTracing? 
+
+Since our policy designer allows you to keep your API policies separate, you can effortlessly control performance and delivery every step of the way with Gravitee’s approach to distributed tracing. This level of independence on API policies helps minimize the chance of over-exposing APIs while maintaining a unified policy approach, simplifying future API deployment, and modernizing your overall API strategy. 
+
+So, if you're looking for a way to simplify debugging, improve monitoring, and enhance visibility into requests across multiple services, our OpenTracing solution with Jaeger as a tracer has you covered. Learn how to set up OpenTracing with the instructions below. 
 
-Enabling OpenTracing gives you the ability to use an open-source tracing system for your microservices. In our case, we've implemented this feature using the open source option, Jaeger. 
 
 == Prerequisites
+In our example, we've employed the usual Gravitee APIM dependencies (MongoDB and Elasticsearch). Need help setting up dependencies? Click the links below. 
+
 
-I need to double check this part with Yann. 
+. MongoDB - for instructions on how to install MongoDB, click here. 
 
-== How To Enable OpenTracing 
+. Elasticsearch - for instructions on how to install Elasticsearch, click here. 
 
-In our examples, we'll use a unmodified configuration file and also provide a configuration file with evnironment variables...(check on this, tiffany)
 
-1. You must enable the tracing feature in `gravitee.yml`. Search for tracing and change `enabled` from `false` to `true`. 
+== How To Enable OpenTracing (Step-By-Step)
+
+=== 1. Enable OpenTracing 
+Enable OpenTracing in the `gravitee.yml` by searching for tracing, then change `enabled` from `false` to `true`. 
 image:apim/3.x/how-tos/enable-opentracing/opentracing-enable-open-tracing-value.png[Change value from false to true.]
+
+=== 2. Start The Jaeger Component 
+Start the Jaeger component with the Docker image by running this command:  
+
+`docker run -d --name jaeger \` 
+
+NOTE: This is the docker run command for installing Jaeger direct from link:/https://www.jaegertracing.io/docs/1.25/getting-started/[Jaegertracing.io] and no customizations to the configuration are included. 
+
+*PRO-TIP:* By by running this Docker command, you'll also get access to the JaegerUI. 
+
+=== 3. Confirm Jaeger Is Enabled On Docker Desktop 
+You can confirm Jaeger has been enabled and is running by checking Docker desktop. 
```

Comment:
Docker desktop is maybe too precise. 
Some people do not have this, and will prefer to check with `docker ps`